provider/online_test.go

```diff
@@ -293,6 +294,47 @@ func testEntityNotFound(t *testing.T, store OnlineStore) {
 	}
 }
 
+func testMassTableWrite(t *testing.T, store OnlineStore) {
+	for i := 0; i < 10; i++ {
```

Comment:
What's the purpose of this for loop?

Reply:
removed. Was to check if my local environment was just weirdly slow